WooCommerce/Classes/ViewRelated/Themes/ThemesPreviewView.swift

```diff
@@ -0,0 +1,146 @@
+
+                VStack {
+                    Button(Localization.startWithThemeButton) {
+                        /* todo */
+                    }
+                    .buttonStyle(PrimaryButtonStyle())
+
+                    Text(String(format: Localization.themeName, "Tsubaki"))
+                        .secondaryBodyStyle()
+                }.padding(Layout.footerPadding)
+            }
+            .toolbar {
+                ToolbarItem(placement: .navigationBarLeading) {
+                    Image(uiImage: .closeButton)
+                }
+
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    Menu {
+                        ForEach(PreviewDevice.allCases, id: \.self) { device in
+                          menuItem(for: device)
+                        }
+                    } label: {
+                         Image(systemName: "macbook.and.iphone")
+                            .bodyStyle()
+                    }
+                }
+            }
+            .navigationBarTitleDisplayMode(.inline)
+        }
+    }
+
+    private func menuItem(for device: PreviewDevice) -> some View {
+        Button {
+            self.selectedDevice = device
+        } label: {
+            Text(Localization.getMenuTitle(for: device))
+            if self.selectedDevice == device {
+                Image(systemName: "checkmark")
+            }
+        }
+    }
+}
+
+private extension ThemesPreviewView {
+    private enum Layout {
+        static let toolbarPadding: CGFloat = 16
+        static let dividerHeight: CGFloat = 1
+        static let footerPadding: CGFloat = 16
+    }
+
+    private enum Localization {
+        static let menuMobile = NSLocalizedString(
+            "themesPreviewView.menuMobile",
+            value: "Mobile",
+            comment: "Menu item: mobile"
+        )
+        static let menuTablet = NSLocalizedString(
+            "themesPreviewView.menuTablet",
+            value: "Tablet",
+            comment: "Menu item: tablet"
+        )
+
+        static let menuDesktop = NSLocalizedString(
+            "themesPreviewView.menuMobile",
+            value: "Desktop",
+            comment: "Menu item: desktop"
+        )
+        static let startWithThemeButton = NSLocalizedString(
+            "themesPreviewView.startWithThemeButton",
+            value: "Start with This Theme",
+            comment: "Button in theme preview screen to pick a theme."
+        )
+        static let themeName = NSLocalizedString(
+            "themesPreviewView.themeName",
+            value: "Theme: %@",
+            comment: "Name of the theme being previewed."
+        )
+
+        static func getMenuTitle(for device: PreviewDevice) -> String {
+            switch device {
+            case .desktop:
+                return menuDesktop
+            case .tablet:
+                return menuTablet
+            case .mobile:
+                return menuMobile
+            }
+        }
```

Comment:
For readability, please add a computed variable in `PreviewDevice` instead of this static method:

```Swift
var menuTitle: String {
    switch self {
    ...
    }
}
```

You can still use the localized string here by calling ThemesPreviewView.Localization.

Reply:
Makes sense, added in https://github.com/woocommerce/woocommerce-ios/pull/11452/commits/4ac37b33da8de0261738e44536fbef7070f32ab6